projects/cloud/app/frontend/components/Home.tsx

```diff
@@ -270,20 +278,34 @@ const Home = () => {
     </Modal>
   );
 
+  const { accountName: organizationName } = useParams();
+
+  const client = useApolloClient();
+  const [homeStore] = useState(new HomeStore(client));
+
+  useEffect(() => {
+    if (!organizationName) {
+      return;
+    }
+    homeStore.load(organizationName);
+  }, [organizationName]);
+
```

Comment:
I'd consider using the [React Mobx](https://mobx.js.org/react-integration.html) integration that will take care of binding the component to observable state in the store.

Reply:
this is the other way around - we need to reload store when the state of the component changes. In this case, the state is the URL from which we get the `organizationName`. I'm not sure if there's a cleaner way to achieve this.